.github/workflows/container-image.yml

```diff
@@ -182,3 +176,49 @@ jobs:
             ghcr.io/${{ github.repository_owner }}/policy-server@${{ steps.build-latest.outputs.digest }}
         env:
           COSIGN_EXPERIMENTAL: 1
+
+      - name: Install the bom command
+        uses: kubewarden/github-actions/kubernetes-bom-installer@v1
+
+      - name: Create directory to store container SBOM files
+        shell: bash
+        run: |
+          mkdir policy-server-container-image-sbom
+
+      - name: Create SBOM file for the container  image
+        if: ${{ startsWith(github.ref, 'refs/heads/') }}
+        shell: bash
+        run: |
+          set -e
+          bom generate -n https://kubewarden.io/ --image ghcr.io/${{ github.repository_owner }}/policy-server@${{ steps.build-latest.outputs.digest }} -o policy-server-container-image-sbom/policy-server-container-image-sbom.spdx
+
+      - name: Create SBOM file for the container  image
+        if: ${{ startsWith(github.ref, 'refs/tags/') }}
```

Comment:
I think this step and the previous one can be collapsed into one, and no `if ${{ startsWith(github.ref ... ` is needed.
This is because the workflow is run for `on.push.branches["main"]` and `on.push.tags[ "v*" ]` which I think is equivalent.

Reply:
Yes, I think you are right. We could have one single "Create SBOM file for the container image" step, without any `if` statement.

Is that what you mean?

Reply:
yep, correct.